docs/content/docs/getting_started/discover_heimdall.adoc

```diff
@@ -183,11 +183,11 @@ If you let heimdall care about most of the existing authentication and authoriza
 
 ....
 
-This way you let heimdall orchestrate existing authentication and authorization systems and also provide the information about the authenticated and authorized subjects to your microservice in a unified format completely independent of the used authentication strategy or protocol.
+This way you let heimdall orchestrate existing authentication and authorization systems and also provide the information about the authenticated and authorized subjects to your microservice in a unified format. Completely independent of the used authentication strategy or protocol.
```

Comment:
The added dot makes the new second sentence feel incomplete, at least from my perspective.